mingw-w64-scc/PKGBUILD

```
@@ -0,0 +1,41 @@
+# Maintainer: Wu Zhenyu <wuzy01@qq.com>
+
+realname=scc
```

It's better to start this variable with an underscore (`_`) and change its reference. Since Archlinux doesn't define this variable it's better to mark it private as it's done on the templates.
```suggestion
_realname=scc
```